client/src/common/CartCard.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Card,
+  CardActions,
+  CardContent,
+  CardActionArea,
+  CardMedia,
+  IconButton,
+  Typography,
+  Box,
+  ButtonGroup,
+  keyframes,
+} from '@mui/material';
+
+import AttachMoneyIcon from '@mui/icons-material/AttachMoney';
+import AddIcon from '@mui/icons-material/Add';
+import RemoveIcon from '@mui/icons-material/Remove';
+import { useInView } from 'react-intersection-observer';
+import React from 'react';
+import { TypeClothes } from './ProductCard';
+
+type CardProps = {
+  product: TypeClothes;
+};
+
+const CartCard: React.FC<CardProps> = ({ product }) => {
+  const { ref, inView }: any = useInView();
+
+  const CartCardAnimation = keyframes`
+  0% {
+    transform: scale(0.9)
+  }
+  100% {
+    transform: scale(1)
+  }
+  `;
+
+  const style = {
+    cardStyle: {
+      display: 'flex',
+      width: '100%',
+      m: '2%',
+      backgroundColor: 'primary.main',
+      flexDirection: 'column',
+      alingContent: 'space-between',
+      animation: inView && `${CartCardAnimation} 0.25s both`,
```

Comment:
@Franchos no da algun error en consola cuando se renderiza la animación? porque le llega `false` a veces por eso

Reply:
A mi me daba error esta linea. Tendrias que modificarla 
```suggestion
      animation: inView ? `${CartCardAnimation} 0.25s both` : '',
```